astropy/utils/console.py

```diff
@@ -1102,7 +1102,7 @@ def print_code_line(line, col=None, file=None, tabwidth=8, width=70):
 
     if col is not None:
         if col >= len(line):
-            raise ValueError("col must be less the the line length.")
+            raise ValueError("col must be less the line length.")
```

Comment:
Just flagging this error message change for anyone reviewing. We somewhat consider the error message as part of the api.

Reply:
Since this is a necessary change for a real typo I would agree to the change with a 6.0 milestone. Except that it should be `must be less than the line length`.

Reply:
I will remove this change from my PR then and let you decide when to fix the error message as it is part of the API. Is that correct?

Reply:
@taldcroft is suggesting to implement the typo fix and to change the message.